packages/trader/src/Utils/Validator/declarative-validation-rules.js

```diff
@@ -21,9 +21,10 @@ export const validPassword = value => /(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]+
 export const validLetterSymbol = value => !/[`~!@#$%^&*)(_=+[}{\]\\/";:?><,|\d]+/.test(value);
 const validGeneral = value => !/[`~!@#$%^&*)(_=+[}{\]\\/";:?><|]+/.test(value);
 export const validAddress = value => !/[`~!$%^&*_=+[}{\]\\"?><|]+/.test(value);
-export const validPostCode = value => /^[a-zA-Z\s\W\d-]*$/.test(value);
-export const validPhone = value => /^\+(?:[0-9] ?){6,14}[0-9]$/.test(value);
-export const validCountryCode = (list, value) => list.some(item => value.startsWith(`+${item.phone_idd}`));
+export const validPostCode = value => /^[-A-Za-z0-9\s]{0,20}$/.test(value);
+export const validPhone = value => /^\+?((-|\s)*[0-9]){8,35}$/.test(value.replace(/[ ]/g, ''));
+export const validCountryCode = (list, value) =>
```

Comment:
Perhaps a suggestion for the future, but should we refactor these functions and prefix them with `is` as per our styling guide? i.e. `isValidPostCode` and `isValidPhone`